.github/workflows/_build.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: Build
+on:
+  workflow_call:
+    inputs:
+      image:
+        description: "Output image to publish"
+        type: string
+        default: "ghcr.io/krystal/actions-runner:latest"
+      runner_version:
+        description: "Runner version"
+        type: string
+        default: "2.315.0" # https://github.com/actions/runner/releases
+      docker_compose_version:
+        description: "Docker compose version"
+        type: string
+        default: "2.26.1" # https://github.com/docker/compose/releases
```

Comment:
I decided to hard-code the latest versions here, meaning we need to periodically bump them.

 I figured this would make for a more stable runner, but it would also avoid any issues with `latest` tags not being latest due to Docker build cache.